packages/mdc-menu/simple/foundation.js

```diff
@@ -399,69 +335,202 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
     }, numbers.SELECTED_TRIGGER_DELAY);
   }
 
-  /** @private */
-  autoPosition_() {
-    if (!this.adapter_.hasAnchor()) {
-      return;
-    }
+  /**
+   * @return {AutoLayoutMeasures} Measurements used to position menu popup.
+   */
+  getAutoLayoutMeasures_() {
```

Comment:
Maybe rename "Measures" to "Measurements" throughout?

Reply:
done